website/docs/providers/type/community-index.html.markdown

```diff
@@ -29,7 +29,7 @@ please fill out this [community providers form](https://docs.google.com/forms/d/
 - [Apigee](https://github.com/zambien/terraform-provider-apigee)
 - [Artifactory](https://github.com/atlassian/terraform-provider-artifactory)
 - [Auth](https://github.com/Shuttl-Tech/terraform-provider-auth)
-- [Auth0](https://github.com/alexkappa/terraform-provider-auth0)
+- [Auth0](https://github.com/delphire/terraform-provider-auth0)
```

Comment:
These two appear to be separate projects without a common lineage. I'm not against adding the delphire one, but please leave the alexkappa one in place.

Reply:
Ah, I see. Looks like this one changed since my original PR, and when I rebased, I kept my change. It used to be https://github.com/bocodigitalmedia/terraform-provider-auth0, which redirects to https://github.com/delphire/terraform-provider-auth0. @cgriggs01 changed it to https://github.com/alexkappa/terraform-provider-auth0 about a week ago (see 2010ce28047c6ef01384ea301e117df2fdef5a79). I will remove that one from my PR.

Reply:
7be28cf1e0b62ff1268fe6adc2a840a450976f27 removes the change to the auth0 provider.

Reply:
Ah, I see. Looks like this one changed since my original PR, and when I rebased, I kept my change. It used to be https://github.com/bocodigitalmedia/terraform-provider-auth0, which redirects to https://github.com/delphire/terraform-provider-auth0. @cgriggs01 changed it to https://github.com/alexkappa/terraform-provider-auth0 about a week ago (see 2010ce2). I removed it from the PR.